recipes/apipkg/meta.yaml

```diff
@@ -0,0 +1,61 @@
+package:
+  name: apipkg
+  version: "1.4"
+
+source:
+  fn: apipkg-1.4.tar.gz
+  url: https://pypi.python.org/packages/32/37/6ce6dbaa8035730efa95e60b09498ec17000d137742391ff46974d9ef859/apipkg-1.4.tar.gz
+  md5: 17e5668601a2322aff41548cb957e7c8
+#  patches:
+   # List any patch files here
+   # - fix.patch
+
+# build:
```

Comment:
The commented-out boilerplate can be removed from the various PyPi recipes in this PR.